frontend/src/utils/random_data.ts

```diff
@@ -0,0 +1,69 @@
+export const NICKNAMES = [
+  "Person_A",
+  "Person_B",
+  "Person_C",
+  "Person_D",
+  "Person_E",
+  "Person_F",
+  "Person_G",
+  "Person_H",
+  "Person_I",
+  "Person_J",
+  "Person_K",
+  "Person_L",
+  "Person_M",
+  "Person_N",
+  "Person_O",
+  "Person_P",
+  "Person_Q",
+  "Person_R",
+  "Person_S",
+  "Person_T",
+  "Person_U",
+  "Person_V",
+  "Person_W",
+  "Person_X",
+  "Person_Y",
+  "Person_Z",
+]
+
+export const PRONOUNS = [
+  "He/Him",
+  "She/Her",
+  "They/Them",
+  "Xe/Xem",
+  "Ze/Zem",
+]
```

Comment:
```suggestion
];
```
Please add semicolon at end of line.